# ParasiteGenomics
A [tutorial](https://github.com/GeertsManon/ParasiteGenomics/wiki) on how to detect CNV in genomic data of parasites


## Important Notes

- Throughout the tutorial, highlighted terms link to additional information or definitions. Make sure to check these resources if you're unfamiliar with any concepts.
- You'll find questions ‚ùì at the end of each section. These questions are designed to test your understanding of the concepts and analyses you've just performed. Take time to think about and answer these questions before moving on to the next section.
- Questions marked with üí° have hints available if you're stuck.
- For a deeper understanding of the biological significance of copy number variations in Leishmania, refer to the [original publication](https://www.pnas.org/doi/full/10.1073/pnas.1920136117).

## Time Schedule



This tutorial is designed to be completed in approximately 2 hours and 15 minutes:

- [Set up your environment](https://github.com/GeertsManon/ParasiteGenomics/wiki/Introduction)

- Upstream Analyses (Exercises 1-3): 45 minutes

  - Setting up the environment
  - [Exploring the reference genome](https://github.com/GeertsManon/ParasiteGenomics/wiki/Exercise-1:-Exploring-the-reference-genome)
  - [Exploring the FASTQ files](https://github.com/GeertsManon/ParasiteGenomics/wiki/Exercise-2:-Exploring-the-FASTQ-files)
  - [Mapping reads to the reference genome](https://github.com/GeertsManon/ParasiteGenomics/wiki/Exercise-3:-Mapping-reads-to-the-genome-using-BWA)


- Downstream Analyses (Exercises 4-5): 1 hour 30 minutes

  - [Chromosome copy number estimation](https://github.com/GeertsManon/ParasiteGenomics/wiki/Exercise-4:-Estimating-chromosome-copy-number-variations)
  - [Local copy number variation detection](https://github.com/GeertsManon/ParasiteGenomics/wiki/Exercise-5:-Estimating-local-copy-number-variations)
  - Data visualization and interpretation
